app/views/staff/session_formats/create.js.erb

```diff
@@ -11,4 +11,5 @@
   $('#config-modal .errors').append('<%=  session_format.errors.full_messages.join(", ") %>');
```

Comment:
Leave name blank, and repeatedly click save. The previous errors don't get cleared but are appended to the previous errors.